modules/network/dhcp-1/python/src/grpc_server/radvd_server.py

```diff
@@ -22,29 +22,42 @@
 class RADVDServer:
   """Represents the RADVD Server"""
 
-  def __init__(self):
+  def __init__(self, enabled=True):
     global LOGGER
-    LOGGER = logger.get_logger(LOG_NAME, 'dhcp-1')
+    LOGGER = logger.get_logger(LOG_NAME, 'dhcp-2')

```

Comment:
@jhughesbiot Is this supposed to be dhcp-2 in the dhcp-1 module?

Reply:
Yes, this has been updated.